src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pydantic.py

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import exec_statement, collect_submodules
+
+# By default, pydantic from PyPi comes with all modules compiled as
+# cpython extensions, which seems to  prevent the pyinstaller from
+# automatically picking the submodules
+is_compiled = exec_statement(
+    """
+        import pydantic
+        if pydantic.compiled:
+            print('\\nTrue')
+        else:
+            print('\\nFalse')
+    """
+).split()[-1] == 'True'
```

Comment:
There's a `get_module_attribute()` hook utility:
```suggestion
is_compiled = get_module_attribute('pydantic', 'compiled') == 'True'
```
Obviously you'd need to import it.

Reply:
Thanks!